server/hedley/modules/custom/hedley_general/css/date-selector.css

```diff
@@ -0,0 +1,172 @@
+	 background-color: #e6e6e6;
+	 color: #fff;
+}
+
+.date-selector-popup .days table td.date-selector--selected, .ui.form .date-selector-popup .days table td.date-selector--selected {
+	 background-color: #000;
+	 color: #fff;
+}
+
+.date-selector-popup .days table td.date-selector--selected p, .ui.form .date-selector-popup .days table td.date-selector--selected p {
+	 border: 3px solid #fff;
+	 border-radius: 6px;
+	 line-height: 64px;
+}
+
+.date-selector-popup .ui.button, .ui.form .date-selector-popup .ui.button {
+	 background: #fff;
+	 border-radius: 5px;
+	 color: #808080;
+	 display: block;
+	 margin-left: 30px;
+	 margin-top: 30px;
+	 width: 90%;
+}
+
+.date-selector-popup .ui.button.save {
+	font-family: Nunito, Arial, sans-serif;
+	font-size: 1.52rem;
+	font-weight: 800;
+	margin: auto;
+	padding-top: 1em;
+	padding-bottom: 1em;
+	text-align: center;
+	text-transform: uppercase;
+}
+
+.date-selector { font-size: 1em; width: 27em; height: 20em; line-height: 1em; background: #eeeeee; color: #333333; cursor: default; -webkit-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none; }
+
+.date-selector > div { float: left; height: 18em; overflow-y: auto; overflow-x: hidden; padding: 1em 0; border-left: 1px solid rgba(0, 0, 0, 0.1); margin-left: -1px; }
+.date-selector > div:nth-child(1) { width: 4.5em; border-left: none; margin-left: 0; }
+.date-selector > div:nth-child(2) { width: 4em; }
+.date-selector > div:nth-child(3) { width: 18.5em; }
+
+.date-selector ol { margin: 0; padding: 0; list-style: none; }
+.date-selector li { margin: 0; padding: 0.25em 1em; }
+
+.date-selector table { table-layout: fixed; border-collapse: collapse; }
+.date-selector th,
+.date-selector td { width: 1.5em; height: 1em; text-align: right; padding: 0.75em 0.5em; }
+.date-selector th { position: relative; font-weight: normal; padding-top: 0.25em; color: #aaaaaa; }
+.date-selector th::after { position: absolute; bottom: 0; left: 0; right: 0; display: block; content: ""; border-bottom: 1px solid rgba(0, 0, 0, 0.1); }
+.date-selector th:first-child,
+.date-selector td:first-child { padding-left: 1em; }
+.date-selector th:last-child,
+.date-selector td:last-child { padding-right: 1em; }
+
+.date-selector li:hover,
+.date-selector td:hover { background: rgba(0, 0, 0, 0.05); }
+
+.date-selector--selected,
+.date-selector--selected.date-selector--selected:hover { background: #69b3ff; color: #ffffff; }
+
+.date-selector--dimmed { color: #aaaaaa; }
+
+.date-selector--disabled,
+.date-selector--disabled.date-selector--disabled:hover { background: transparent; color: #dddddd; }
+
+/* scrollable-year */
+.date-selector--scrollable-year { width: 28em; }
+.date-selector--scrollable-year > div:nth-child(1) { width: 5.5em; }
+
+
+/*
+Dropdown
```

Comment:
Does this comment help the understanding? Either let's remove it or add more explanation.

Reply:
Same as https://github.com/TIP-Global-Health/eheza-app/pull/1177#discussion_r1646641701